addons/test_website/static/tests/tours/custom_snippets.js

```diff
@@ -46,26 +47,37 @@ tour.register('test_custom_snippet', {
         content: "save custom snippet",
         trigger: ".snippet-option-SnippetSave we-button",
     },
-    {
-        content: "confirm save name",
-        trigger: ".modal-dialog button span:contains('Save')",
-    },
     {
         content: "confirm reload",
         trigger: ".modal-dialog button span:contains('Save and Reload')",
     },
-    {
-        content: "go to blocks",
-        trigger: ".o_we_add_snippet_btn",
-    },
     {
         content: "ensure custom snippet appeared",
         trigger: "#oe_snippets .oe_snippet[name='Custom Banner']",
-        run: function() {}, // check
+        run: function() {
```

Comment:
Your linter is not configured properly, it should warn you about a missing space here